src/Services/Catalog/Catalog.API/Extensions/RedisCacheExtensions.cs

```diff
@@ -0,0 +1,90 @@
+﻿using System.IO;
+
+using System.Threading.Tasks;
+using System.Runtime.Serialization.Formatters.Binary;
+
+using Microsoft.Extensions.Caching.Distributed;
+
+namespace Catalog.API.Extensions
+{
+    public static class RedisCacheExtensions
+    {
+        private static IDistributedCache distributedCache;
+        private static IDistributedCache DistributedCache
+        {
+            get { return distributedCache; }
```

Comment:
I don't see this private variable used inside of the class. Does it have an alternative purpose?

Reply:
indeed a property is enough.